cmd/dagger/module.go

```diff
@@ -161,8 +165,9 @@ var moduleInitCmd = &cobra.Command{
 This creates a dagger.json file at the specified directory, making it the root of the new module.
 
 If --sdk is specified, the given SDK is installed in the module. You can do this later with "dagger develop".
+If --blueprint is specified, the given blueprint is installed in the module.
 `,
-	Example: "dagger init --sdk=python",
```

Comment:
nit: not sure if it makes sense to remove `--sdk` from the example help text entirely vs. leaving it and adding `--blueprint`